charts/vault/templates/vault-rb.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if .Values.plugins.readKuberenetsSecrets.namespaces }}
+{{- $releaseNamespace := .Release.Namespace -}}
+{{- range $ns := .Values.plugins.readKuberenetsSecrets.namespaces }}
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: "vault-rb-{{ $ns }}"
```

Comment:
Make the name reflect that this is for the Kubernetes secrets plugin. No need for the namespace in the name, it's namespace scoped.

Reply:
Thanks will do that